Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -1155,9 +1155,9 @@ private void ReconcilePointers()
 
                 foreach (var pointerData in pointers.Values)
                 {
-                    if (pointerData.Pointer is IMixedRealityNearPointer)
+                    if (pointerData.Pointer is IMixedRealityNearPointer nearPointer)
                     {
-                        if (pointerData.Pointer.IsInteractionEnabled)
+                        if (nearPointer.IsInteractionEnabled || nearPointer.IsNearObject)
```

Comment:
What does this change do? I think most (all of the ones I looked at, at least) of our near pointers use `IsNearObject` as part of their `IsInteractionEnabled` checks. Do we care if a near pointer is near an object but not able to interact with it due to some other part of the `IsInteractionEnabled` check? My gut says we wouldn't, but maybe I'm missing something

Reply:
the near pointer's we have are the sphere pointer and the poke pointer. IsNearObject is used to toggle between near and far interactions, which is slightly different from IsInteractionEnabled. As such, the NumNearPointersActive count should go up if the IsNearObject flag is true. 

The bug this addresses is when an articulated hand spawns near an object, but not quite in the range to trigger IsInteractionEnabled. In these instances, the gazecursor would remain on the screen, even though the cursor won't be on the screen if you spawn the hand far away from the object and then move closer to it